apps/wing-console/console/server/src/utils/createRouter.ts

```diff
@@ -76,6 +76,13 @@ export interface FileLink {
   column?: number;
 }
 
+export interface RouterMeta {
+  analytics?: {
+    action?: string;
+    resource?: string;
+  };
+}
+
```

Comment:
Does it make sense to allow undefined `action` and `resource` if `analytics` is defined? I think this better:

```suggestion
export interface RouterMeta {
  analytics?: {
    action: string;
    resource: string;
  };
}

```